.github/workflows/ci.yaml

```diff
@@ -94,17 +95,6 @@ jobs:
          micromamba list
          cat /proc/cpuinfo || (/usr/sbin/system_profiler SPHardwareDataType; /usr/sbin/sysctl -a | grep machdep.cpu)
 
-    - name: Install pytest and plugins
-      run: |
-         micromamba install pytest pytest-pep8 pytest-cov codecov
-
-    - name: Install GROMACS (${{ matrix.gromacs-version }})
```

Comment:
update `name: micromamba environment and package installation (Python ${{ matrix.python-version }})` to also include GROMACS version, along the lines of 

```yaml
name: micromamba package and testing environment installation (Python ${{ matrix.python-version }}, GROMACS ${{ matrix.gromacs-version }})

Reply:
good catch, thanks – updated